### ISSUE TITLE
Feature Request: Fingerprint Unlock

### ISSUE THREAD

**Issue by KTSCode** [vishesh/sealnote#32]
I know that the fingerprint scanner isn't the most secure method of authentication, but I think it would make this app significantly better. Having to type your password in all the time, make this app painful to use. To increase security, you could require that the user enter their password, every time the phone is restarted.

**Comment by vishesh** [comment]
Thanks. @akshatgoel wanna take a shot at his?

**Comment by akshatgoel** [comment]
Although this is cool to have but I have a different opinion on this. Currently, having a separate password for Sealnote gives us 2-factor authentication for password protected devices.
Moving it to fingerprint unlock will result in single-factor authentication.

I have not worked much with fingerprint-unlocked devices so if you have any idea on how people work with it, we can think of a good approach to tackle this. Thoughts?

**Comment by vishesh** [comment]
@akshatgoel You are right actually. It offers convenience but kind of defeats the purpose of second password. The password will have to persist in disk, which makes this not ideal. We don't save password in bundle right? It stays in memory and erases with time? 

On other hand, API wise, seems like even though its for newer Android, we may not have to drop support for older versions. https://developer.android.com/about/versions/marshmallow/android-6.0.html
